### DISCUSSION TITLE
Build Errors with nRF52840DK

### DISCUSSION THREAD

**Discussion by ChanceCuddeback** [zephyrproject-rtos/zephyr#35645] [Q&A] [answered]
Hello all! I'm new to embedded development (Just Arduino experience) so I may be missing a crucial step. I installed Zephyr by following the /latest/getting_started/ in the docs for MacOS (Big Sur 11.4) using the GNU Arm Embedded toolchain (gcc-arm-none-eabi-10-2020-q4-major). I was recently playing around with Nordics NCS software and I was able to load zephyr examples. But I wanted to get away from IDE's like SES to learn more about embedded dev.

I am having trouble building the blinky sample using:
west build -p auto -b nrf52840dk_nrf52840 samples/basic/blinky

I suspect that I have to modify the path to my toolchain (I'm not sure how)?

Would anyone be willing to help me get started with Zephyr. Attached is the call stack and error log. Thank you!

<img width="931" alt="Screen Shot 2021-05-25 at 12 48 59 PM" src="https://user-images.githubusercontent.com/45518430/119559490-9a17fd80-bd57-11eb-8c8f-2ae95a498cf6.png">

**Comment by ChanceCuddeback** [comment, marked as answer]
I fixed it! I reinstalled the GNU toolchain and updated the path, I hadn't included "".  Now on to flashing. I would delete this post but I don't know how. 

Because it's up, do any of you have some good resources (other than the docs) for learning the basics of Zephyr.

**Comment by bsingh19** [comment]
Getting started with Zephyr (Setup toolchain, compilation, board bring-up, Hello World example)!!

Systems: MacBook
Board : nRF52833 DK (Optional)

https://youtu.be/4emoE-8hsqg
